### PR TITLE
jfrparser-cli: add cli that formats json

### PR DESCRIPTION
**Pull request by tomershafir**

### CODE REVIEW

File: cmd/jfrparser/format/testdata/cortex-dev-01__kafka-0__cpu__0.json.gz
Comment (korniltsev) [inline cmd/jfrparser/format/testdata/cortex-dev-01__kafka-0__cpu__0.json.gz:1]: can you put the file next to the jfr file?

File: cmd/jfrparser/main.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	c.src = args[0]
+	c.target = args[1]
+}
+
+type formatter interface {
+	// Formats the given JFR
+	Format(*parser.Parser) ([]byte, error)
+}
+
+// TODO: mov jfr2pprof here
```
Comment (korniltsev) [inline cmd/jfrparser/main.go:39]: can you add pprof here and remove jfr2pprof ? we don't need two converters

File: cmd/jfrparser/format/format_json.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	if tmp, err = json.Marshal(d); err != nil {
+		return fmt.Errorf("json.Marshal error: %w", err)
+	}
+	f.outBuf = append(f.outBuf, tmp...)
+	return nil
+}
+
+func (f *formatterJson) appendInit(p *parser.Parser) error {
+	var err error
+	f.outBuf = append(f.outBuf, []byte("[{\"Header\":")...)
```
Comment (korniltsev) [inline cmd/jfrparser/format/format_json.go:33]: why is this json building manual?

**Comment by tomershafir** [comment]
@korniltsev addressed comments and more

**Review by korniltsev — APPROVED** [review]

**Pull request merged**